src/components/player/cue-bar.tsx

```diff
@@ -111,9 +111,7 @@ const NoteCue: React.FC<any> = ({
               {note.title}
             </span>
           )}
-          <TruncateMarkup lines={2}>
-            <div>{note.description}</div>
-          </TruncateMarkup>
+          <div className="line-clamp-2">{note.description}</div>
```

Comment:
we have really good support of [webkit-line-clamp](https://caniuse.com/?search=webkit-line-clamp) CSS feature